modules/iam/groups/data.tf

```diff
@@ -0,0 +1,7 @@
+data "aws_iam_policy" "admin_policy" {
```

Comment:
What about the name as just `admin` or `admin_access` to avoid repeating the policy part

https://nimblehq.co/compass/development/code-conventions/terraform/#convention-6

Same for below if agree

Reply:
Nice one, updated in [63588ef](https://github.com/nvminhtue/tyson_devops_ic/pull/23/commits/63588ef1acd03c08437d8e5cc663d62103c3d058) alongside with other positioning adjustments